crates/kitsune_p2p/kitsune_p2p/src/spawn/actor/fetch/mod.rs

```diff
@@ -0,0 +1,5 @@
+mod fetch_task;
```

Comment:
[The `mod.rs` naming is no longer the recommendation](https://doc.rust-lang.org/reference/items/modules.html#module-source-filenames)

Reply:
I can make a `fetch.rs` a level up. I'm not sure that's the best layout either. This crate probably needs some re-organisation at some point soon anyway